test/test_utils/ecr.py

```diff
@@ -127,6 +174,33 @@ def get_ecr_image_scan_results(ecr_client, image_uri, minimum_vulnerability="HIG
     return scan_findings
 
 
+def get_all_ecr_enhanced_scan_findings(ecr_client, image_uri):
+    """
+    Get list of all vulnerabilities from ECR ENHANCED image scan results
+    :param ecr_client:
+    :param image_uri:
+    :return: list<dict> Scan results
+    """
+    scan_info_findings = get_all_ecr_image_scan_results_using_pagination(
+        ecr_client, image_uri, scan_info_finding_key="enhancedFindings"
+    )
+    scan_findings = []
+    for scan_info_finding in scan_info_findings:
+        scan_findings += [finding for finding in scan_info_finding]
+
+    return scan_findings
+
+
+def ecr_json_serializer(obj):
```

Comment:
Just to confirm, does this function get called for every object to be serialized by the `json.dump` call, or does it only get called for those objects that cannot be serialized?

Reply:
Removed the method altogether.